packages/image_picker/image_picker_android/android/src/test/java/io/flutter/plugins/imagepicker/ImageResizerTest.java

```diff
@@ -50,6 +53,8 @@ public void setUp() throws IOException {
     mockCloseable = MockitoAnnotations.openMocks(this);
     imageFile = new File(getClass().getClassLoader().getResource("pngImage.png").getFile());
     svgImageFile = new File(getClass().getClassLoader().getResource("flutter_image.svg").getFile());
+    tallJPG = new File(getClass().getClassLoader().getResource("jpgImageTall.jpg").getFile());
```

Comment:
Could you stick a comment on each of these saying what the dimensions are? I think it would be useful for understanding the tests.